tests/store/tracking/test_file_store.py

```diff
@@ -282,6 +282,10 @@ def test_search_experiments_filter_by_tag(self):
             ("exp3", [ExperimentTag("k e y", "value")]),
         ]
         for name, tags in experiments:
+            # sleep for windows file system current_time precision in Python to enforce
+            # deterministic ordering based on last_update_time (creation_time due to no
+            # mutation of experiment state)
+            time.sleep(0.01)
```

Comment:
On windows, `time.sleep(0.001)` results in 15~16 ms sleep, so `time.sleep(0.001)` should work.

Reply:
Sorry about that. I couldn't remember if it was 0.01 or 0.001 that you had put on that closed branch. This will be updated in the next commit :)

Reply:
It was 0.001 :)